### PR TITLE
feat(mc): use all minio instances

### PR DESCRIPTION
**Pull request by jumana-s**
# Description

**What your PR adds/fixes/removes**
Instead of using MinIO Instances that start with "minio", use all MinIO Instances available in /vault/secrets/.

# Tests / Quality Checks

## Automated Testing/build and deployment
- [ ] Does the image pass CI successfully (build, pass vulnerability scan, and pass automated test suite)?
- [ ] If new features are added (new image, new binary, etc), have new automated tests been added to cover these?
- [ ] If new features are added that require in-cluster testing (e.g. a new feature that needs to interact with kubernetes), have you added the `auto-deploy` tag to the PR before pushing in order to build and push the image to ACR so you can test it in cluster as a custom image?

## JupyterLab extensions

- [ ] Are all extensions "enabled" (`jupyter labextension list` from inside the notebook)?

## VS Code tests

- [ ] Does VS Code open?
- [ ] Can you install extensions?

## Code review

- [ ] Have you added the `auto-deploy` tag to your PR before your most recent push to this repo?  This causes CI to build the image and push to our ACR, letting reviewers access the built image without having to create it themselves
- [ ] Have you chosen a reviewer, attached them as a reviewer to this PR, and messaged them with the SHA-pinned image name for the final image to test (e.g. `k8scc01covidacr.azurecr.io/machine-learning-notebook-cpu:746d058e2f37e004da5ca483d121bfb9e0545f2b`)?

### CODE REVIEW

File: output/remote-desktop/start-remote-desktop.sh [hunk 1]
```diff
@@ -67,6 +67,41 @@ if [ -d $RESOURCES_PATH/desktop-files ]; then
     cp /opt/install/desktop-files/.config/xfce4/xfce4-panel.xml $HOME/.config/xfce4/xfconf/xfce-perchannel-xml/
 fi
 
+# Create MinIO Desktop files
+# May need to wait a few seconds for Vault to come up
+n=0
+until [ "$n" -ge 5 ]
```
Comment (bryanpaget) [inline output/remote-desktop/start-remote-desktop.sh:73]: does this need to be wrapped in quotes?

File: output/remote-desktop/start-remote-desktop.sh [hunk 2]
```diff
@@ -67,6 +67,41 @@ if [ -d $RESOURCES_PATH/desktop-files ]; then
...
+
+           # echo "dir not empty"
+           cat <<EOF > $HOME/minio-$(basename $tenant).Desktop
+[Desktop Entry]
+Name=Minio - $(basename $tenant)
+Version=1.0
+Type=Application
+Terminal=false
+Exec= python3 /usr/bin/minio-launch.py ${tenant}.json %F
+Icon=/resources/minio-icon.png
```
Comment (bryanpaget) [inline output/remote-desktop/start-remote-desktop.sh:89]: I always look to /usr/share/icons for icons and if you are old like me you'll also think to look in /use/share/pixmaps!

No need to change anything here. :-P

File: resources/remote-desktop/start-remote-desktop.sh [hunk 3]
```diff
@@ -67,6 +67,41 @@ if [ -d $RESOURCES_PATH/desktop-files ]; then
     cp /opt/install/desktop-files/.config/xfce4/xfce4-panel.xml $HOME/.config/xfce4/xfconf/xfce-perchannel-xml/
 fi
 
+# Create MinIO Desktop files
+# May need to wait a few seconds for Vault to come up
+n=0
+until [ "$n" -ge 5 ]
```
Comment (bryanpaget) [inline resources/remote-desktop/start-remote-desktop.sh:73]: does this need to be wrapped in quotes?

**Comment by blairdrummond** [comment]
@bryanpaget can you look at fixing this up?

**Review by bryanpaget — APPROVED** [review]

**Pull request merged**